### PR TITLE
WIP - Feature/pool asset management

### PR DESCRIPTION
**Pull request by jak-pan**
This is still in progress, could be merged as baseline, but needs testing first.

### CODE REVIEW

**Comment by gleb-urvanov** [comment]
Please introduce Randomness type in xyk::Trait implementation for runtime
runtime/src/lib.rs line 254

**Review by gleb-urvanov — CHANGES_REQUESTED** [review]
Please introduce Randomness type in xyk::Trait implementation for runtime
runtime/src/lib.rs line 254

File: pallets/xyk/src/lib.rs [hunk 1]
```diff
@@ -57,51 +83,187 @@ decl_module! {
             Ok(())
         }
 
-        fn create_pool(origin,
+        /// Creates liquidity pool pair for Asset 1 and Asset 2
+        fn create_liquidity_pool(origin,
```
Comment (gleb-urvanov) [inline pallets/xyk/src/lib.rs:87]: Liquidity pool should always be created with some initial liquidity amount

Comment (jak-pan) [reply pallets/xyk/src/lib.rs:87]: Makes sense, will call mint function.

**Pull request merged**